include/ua_client_subscriptions.h

```diff
@@ -151,7 +151,7 @@ UA_CreateMonitoredItemsResponse UA_EXPORT
 UA_Client_MonitoredItems_createEvents(UA_Client *client,
             const UA_CreateMonitoredItemsRequest request, void **contexts,
             UA_Client_EventNotificationCallback *callbacks,
-            UA_Client_DeleteMonitoredItemCallback *deleteCallback);
+            UA_Client_DeleteMonitoredItemCallback *deleteCallbacks);
```

Comment:
Changes like these should not belong to this PR. Better move them to a separate PR and explain why it is necessary to change this.